vm.erb

```diff
@@ -0,0 +1,92 @@
+<!--
+WARNING: THIS IS AN AUTO-GENERATED FILE. CHANGES TO IT ARE LIKELY TO BE
+OVERWRITTEN AND LOST. Changes to this xml configuration should be made using:
+  virsh edit test
+or other application using the libvirt API.
+-->
+
+<domain type='kvm'>
+  <name><%= name %></name>
+  <uuid><%= uuid %></uuid>
+  <memory unit='KiB'><%= memory %></memory>
+  <currentMemory unit='KiB'><%= memory %></currentMemory>
+  <vcpu placement='static'><%= cpus %></vcpu>
+  <os>
+    <type arch='x86_64' machine='pc-i440fx-rhel7.0.0'>hvm</type>
+    <boot dev='network'/>
+  </os>
+  <features>
+    <acpi/>
+    <apic/>
+  </features>
+  <cpu mode='custom' match='exact' check='partial'>
+    <model fallback='allow'>Skylake-Client-IBRS</model>
+  </cpu>
+  <clock offset='utc'>
+    <timer name='rtc' tickpolicy='catchup'/>
+    <timer name='pit' tickpolicy='delay'/>
+    <timer name='hpet' present='no'/>
+  </clock>
+  <on_poweroff>destroy</on_poweroff>
+  <on_reboot>restart</on_reboot>
+  <on_crash>destroy</on_crash>
+  <pm>
+    <suspend-to-mem enabled='no'/>
+    <suspend-to-disk enabled='no'/>
+  </pm>
+  <devices>
+    <emulator>/usr/libexec/qemu-kvm</emulator>
+    <interface type='network'>
+      <mac address='<%= mac %>'/>
+      <source network='default'/>
+      <model type='virtio'/>
+      <boot order='1'/>
+      <address type='pci' domain='0x0000' bus='0x00' slot='0x03' function='0x0'/>
+    </interface>
+    <disk type='file' device='disk'>
+      <driver name='qemu' type='qcow2'/>
+      <source file='/home/scorpio/hdd/test.qcow2'/>
```

Comment:
You may need to rethink this one, the VM QCOW image won't be the same for all machines

Reply:
Yeah... we'll need to figure a way to fetch the QCOW image or just remove it flat out

Reply:
Or you could use a combo of the machine's name and IP with `.` replaced with `-`

Reply:
alright, will do